fs/arch/install-to-emmc-middle

```diff
@@ -0,0 +1,75 @@
+# start of install-to-emmc-middle
```

Comment:
Did you test this script?

Reply:
No, waiting on OXIDE docs to test it on my own machine. Otherwise need a tester.